config/routes.rb

```diff
@@ -29,6 +29,7 @@
   resources :checkouts, only: [:index, :create, :show, :update, :destroy] do
     collection do
       patch :return_all
+      get :display_returned
```

Comment:
I'm wondering about making this route to `:index` with a param set so it would add a branch to that action in order to avoid the need for a new action entirely.  Or it may be possible to get away with a special param on `:index` without a new route.  What do you think?

Reply:
I have reworked it so that there is a `display_returned` param that changes the behavior of the `set_checkouts` function, so that the checkouts list displayed on the page is controlled entirely through the `index` action/route.